### PR TITLE
ci: run tests on multiple wagtail versions

### PR DESCRIPTION
**Pull request by ruisaraiva19**
This runs tests on Wagtail version `2.9`, `2.10`, `2.11`, and `2.12`.

### CODE REVIEW

**Comment by zerolab** [comment]
Looks like the tests will need updating:
there's the migration dependency on `0053_locale_model` which came in with 2.11
then there is the `get_document_model_string` import error and I suspect a couple of others relating to deprecations

File: example/home/migrations/0002_create_homepage.py [hunk 1]
```diff
@@ -53,7 +56,7 @@ class Migration(migrations.Migration):
         [
             ("wagtailcore", "0053_locale_model"),
         ]
-        if wagtail.__version__ >= "2.11"
+        if wagtail_version >= (2, 11)
```
Comment (zerolab) [inline example/home/migrations/0002_create_homepage.py:59]: you could import https://github.com/wagtail/wagtail/blob/main/wagtail/__init__.py#L10 (`from wagtail import VERSION as WAGTAIL_VERSION`) and then `if WAGTAIL_VERSION >= (2, 11)

File: example/home/test/test_blog.py [hunk 2]
```diff
@@ -426,8 +427,12 @@ def test_blog_embed(self):
             if block["blockType"] == "VideoBlock":
                 embed = block["youtubeLink"]
                 self.assertTrue(isinstance(embed["url"], str))
-                self.assertEquals(embed["embed"], raw_embed["html"])
-                self.assertEquals(embed["rawEmbed"], json.dumps(raw_embed))
+                if WAGTAIL_VERSION >= (2, 11):
+                    self.assertEquals(embed["embed"], raw_embed["html"])
+                    self.assertEquals(embed["rawEmbed"], json.dumps(raw_embed))
+                else:
+                    self.assertEquals(embed["embed"], None)
```
Comment (zerolab) [inline example/home/test/test_blog.py:434]: question: why is this the case for <= 2.11?

could probably use a comment. future us will thank you ;)

**Review by zerolab — APPROVED** [review]
Other than that one final question, this LGTM

Thank you @ruisaraiva19 !

File: example/home/test/test_blog.py (hunk 2 above)
Comment (ruisaraiva19) [reply example/home/test/test_blog.py]: @zerolab I just remembered [this issue](https://github.com/wagtail/wagtail/issues/6656#issuecomment-749711713) and now I [added](https://github.com/GrappleGQL/wagtail-grapple/pull/176/commits/ff78bb127e6a30c42eb8effd6681bae0e04024d5) a custom `WAGTAILEMBEDS_FINDERS` setting when running with Wagtail version < 2.11 and now the tests pass as expected without the if statement.

**Pull request merged**